From 456efa60127ab42f3099160c58c017625b43115a Mon Sep 17 00:00:00 2001
From: Joey Huab <joey@evolution-x.org>
Date: Thu, 23 Dec 2021 21:44:02 +0100
Subject: [PATCH] sm8250-common: parts: Increase BootCompletedReceiver priority

 * set the priority to SYSTEM_HIGH_PRIORITY to match other system services
   so our settings are restored faster on bootup

Change-Id: I46b6afed62f2e9f166934088b757b7dfd82ddf4b
---
 parts/AndroidManifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/parts/AndroidManifest.xml b/parts/AndroidManifest.xml
index 49f708d..22fe3f9 100644
--- a/parts/AndroidManifest.xml
+++ b/parts/AndroidManifest.xml
@@ -39,7 +39,7 @@
         android:theme="@style/Theme.SubSettingsBase">
 
         <receiver android:name=".BootCompletedReceiver">
-            <intent-filter>
+            <intent-filter android:priority="1000">
                 <action android:name="android.intent.action.BOOT_COMPLETED" />
                 <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
-- 
2.25.1

